.circleci/config.yml

```diff
@@ -98,6 +98,11 @@ jobs:
       - *cmd_restore_cache
       - attach_workspace:
           at: ~/art-backend/tmp
+      - run:
+          name: run migrations
+          command: |
+            pipenv run python manage.py makemigrations
```

Comment:
Why make migrations on circle ci again? All you need to do is run migration that has been made locally.

Reply:
Absence of this was the error that was causing my tests to fail on Circle CI. Once I added this line of code my tests could pass without throwing psycopg2 errors.
reference : https://stackoverflow.com/questions/48184300/when-in-run-test-cases-then-i-will-get-this-error-psycopg2-operationalerror-cu?rq=1